skdownscale/spatial_models/sd.py

```diff
@@ -0,0 +1,124 @@
+import numpy as np
+import pandas as pd
+import xarray as xr
+
+
+class SpatialDisaggregator:
+    """
+    Spatial disaggregation model class
+    Apply spatial disaggregation algorithm to an xarray Dataset with fit
+    and predict methods using NASA-NEX method for spatial disaggregation
+    (see Thrasher et al, 2012).
+
+    Parameters
+    ----------
+    var : str
+          specifies the variable being downscaled. Default is
+          temperature and other option is precipitation.
```

Comment:
minor nit but these are over-indented (same for all docstrings here)

Reply:
I think this is fixed now.